src/data/moves_info.h

```diff
@@ -893,16 +867,10 @@ const struct MoveInfo gMovesInfo[MOVES_COUNT_DYNAMAX] =
         .description = COMPOUND_STRING(
             "Charges the foe with a full-\n"
             "body tackle."),
-        #if B_UPDATED_MOVE_DATA >= GEN_7
-            .power = 40,
-        #elif B_UPDATED_MOVE_DATA >= GEN_5
-            .power = 50,
-        #else
-            .power = 35,
-        #endif
+        .power = CONFIG_BY_GEN(35, GEN_5, 50, GEN_7, 40),
```

Comment:
At a glance, it's not very clear to me what number corresponds to what Gen.

Reply:
Hmm, I assumed reading from left to right would make the most sense. What would you suggest here?

Reply:
I see it simetrically. I don't know if 50 corresponds to `GEN_5` or `GEN_7`, so in this specific cases for configs, they look even more confusing than a double ternary would be.

Reply:
Oh? From the conversation that MGriffin linked above discussing a similar macro that had been proposed last year, the feedback seemed mostly... positive? I didn't have much criticism when discussing my implementation of it either. I suppose it's something that needs to be discussed further...